src/instrument/devices/individual_slits.py

```diff
@@ -0,0 +1,67 @@
+"""
+Individual slits of JJ slit
+"""
+
+import logging
+
+from ophyd import Device
+from ophyd import EpicsMotor
+from ophyd import FormattedComponent as FCpt
+
+logger = logging.getLogger(__name__)
+logger.info(__file__)
+
+
+class IndividualSlits(Device):
+    """
+    Individual Slits to move
+    """
+
+    def __init__(
+        self,
+        prefix: str,
+        h_positive_motor: str,
+        h_negative_motor: str,
+        v_positive_motor: str,
+        v_negative_motor: str,
+        *args,
+        **kwargs,
+    ):
+        """
+        Initializing the prefix and correct suffix format
+        """
+        # Determine the prefix for the motors
+        pieces = prefix.strip(":").split(":")
+        self.motor_prefix = ":".join(pieces[:-1])
+
+        self._h_positive_motor = h_positive_motor
+        self._h_negative_motor = h_negative_motor
+        self._v_positive_motor = v_positive_motor
+        self._v_negative_motor = v_negative_motor
+
+        super().__init__(prefix, *args, **kwargs)
+
+    # Real motors that directly control the slits
+    hp = FCpt(EpicsMotor, "{motor_prefix}:{_h_positive_motor}", labels={"motors"})
```

Comment:
Following a BCDA convention, the `:` is defined by the prefix.  Several suggestions follow:

```suggestion
    hp = FCpt(EpicsMotor, "{motor_prefix}{_h_positive_motor}", labels={"motors"})
```

Reply:
bump